portfolio/src/main/webapp/images.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Food Journal</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <h1>Food Journal</h1>
+      <h2>June 2018-Present</h2>
+      <div class="image-container">
+        <a href="images/sliced_avacado.JPG"><img src="images/sliced_avacado.JPG" alt="Sliced avacado on a plate"/></a>
```

Comment:
This is optional, and would probably make for a good follow-up, rather than necessarily being changed in this pull request. I'm not sure if github has any way to indicate that in the UI. But there, I just indicated it in the comment:

Users not using assistive technologies might also want to know what each dish is (at least I do), so I think it would be nice to have the name/description of the dish below the photo. You can use the `aria-labelledby` attribute on the image tag to link the text to the image rather than repeating it in the `alt` attribute.

Here's some documentation about that property: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute

And Example 1 here shows how it can be used with images: https://www.w3.org/WAI/GL/wiki/Using_aria-labelledby_to_provide_a_text_alternative_for_non-text_content

You may need to play with the layout in order to get the text to be below each image or otherwise laid out so that it's clear what text matches with what image. You might also want to check out the [figure and figcaption](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure) tags which appear to be made for situations like this, though I haven't actually used them and I'm not sure how much browsers let you customize their appearance.

Reply:
just added hover-over captions for the images on this page!